<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_mapexplorer</class>
 <widget class="QWidget" name="dlg_mapexplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>567</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QScrollArea" name="m_pScrollAreaMap">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="m_pLabelWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>366</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
     <widget class="QScrollArea" name="m_pScrollAreaControls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>175</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsControls">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>173</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Voxel at</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pX">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pY">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pZ">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>has Density Value</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pDensity">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>148</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>View</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="m_pLabelCrossSection">
               <property name="text">
                <string>Cross Section    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonCrossSectionX">
               <property name="text">
                <string>X</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupMapping</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonCrossSectionY">
               <property name="text">
                <string>Y</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupMapping</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonCrossSectionZ">
               <property name="text">
                <string>Z</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupMapping</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>17</width>
                 <height>22</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSlider" name="m_pSliderSlice">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_pSpinBoxSlice">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="m_pLabelProjectionAlong">
               <property name="text">
                <string>Projection along</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonProjectionX">
               <property name="text">
                <string>X</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupMapping</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonProjectionY">
               <property name="text">
                <string>Y</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupMapping</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonProjectionZ">
               <property name="text">
                <string>Z</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupMapping</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Display</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonDisplayGamma">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Gamma   </string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupDisplay</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_pSliderGamma">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="m_pSpinBoxGamma">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>0.010000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="m_pRadioButtonDisplayThreshold">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Threshold</string>
               </property>
               <attribute name="buttonGroup">
                <string>m_pButtonGroupDisplay</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_pSliderThreshold">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="m_pSpinBoxThreshold">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QCheckBox" name="m_pCheckBoxGrid">
               <property name="text">
                <string>Grid</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Zoom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_pSliderZoom">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="pageStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="m_pButtonGroupDisplay"/>
  <buttongroup name="m_pButtonGroupMapping"/>
 </buttongroups>
</ui>
